Content.Goobstation.Common/StationReport/StationReportComponent.cs

```diff
@@ -0,0 +1,5 @@
+namespace Content.Goobstation.Common.StationReport;
+
+[RegisterComponent]
+public sealed partial class StationReportComponent : Component;
+//is only used for locating the stationreport
```

Comment:
Put your comment information for this component in a summary block.
```suggestion

/// <summary>
/// your information here
/// </summary>
[RegisterComponent]
public sealed partial class StationReportComponent : Component;
```